scripts/ebounty.lic

```diff
@@ -3037,11 +3046,15 @@ module EBounty
           fput "stand" until standing?
           EBounty.wait_rt
 
-          heirloom = GameObj.loot.find { |obj| obj.name =~ /#{item}/ }
+          if checkright
+            EBounty.get_res("stow ##{GameObj.right_hand.id}", EBounty.data.put_regex)
+          else
```

Comment:
Deysh pointed out that hands are all emptied via `empty_hands` prior to this, so if anything ends up being in checkright then it means that it was grabbed because of having this flag already on. No change needed.